Source/Core/Core/HW/SI/SI_DeviceGCController.cpp

```diff
@@ -164,10 +162,8 @@ GCPadStatus CSIDevice_GCController::GetPadStatus()
 // [00?SYXBA] [1LRZUDRL] [x] [y] [cx] [cy] [l] [r]
 //  |\_ ERR_LATCH (error latched - check SISR)
 //  |_ ERR_STATUS (error on last GetData or SendCmd?)
-bool CSIDevice_GCController::GetData(u32& hi, u32& low)
+bool CSIDevice_GCController::GetData(GCPadStatus& pad_status, u32& hi, u32& low)
```

Comment:
It seems that `pad_status` could be `const` here (and cascading everywhere else) - at least I didn't see anything that would mutate its state.
```suggestion
bool CSIDevice_GCController::GetData(const GCPadStatus& pad_status, u32& hi, u32& low)
```